plugin/helpers.go

```diff
@@ -7,8 +7,11 @@ import "github.com/mattermost/mattermost-server/model"
 
 type Helpers interface {
 	// EnsureBot either returns an existing bot user matching the given bot, or creates a bot user from the given bot.
-	// Returns the id of the resulting bot.
-	EnsureBot(bot *model.Bot) (string, error)
+	// Returns the id of the resulting bot. A profile image or icon image may be optionally passed in to be set for
+	// the existing or newly created bot.
```

Comment:
You had great documentation in place already. Please add it back :wink: 
```suggestion
	// the existing or newly created bot. Returns the id of the resulting bot.
	// A profile image or icon image may be optionally passed in to be set for
    // the existing or newly created bot.
```